src/eloqua/eloqua.js

```diff
@@ -16,7 +16,7 @@ var _elqQ = _elqQ || [];
     require.config({
         paths: { 
             elq: "//img.en25.com/i/elqCfg.min",
-            CerosSDK: "//sdk.ceros.com/standalone-player-sdk-v3"
+            CerosSDK: '//sdk.ceros.com/standalone-player-sdk-v5'
```

Comment:
Technically single or double quotes don't have a difference, but I like to keep consistency on the rest of the file, so if the file is already using double quotes I would use that.

Reply:
I've updated the single quotes to double quotes, committed and pushed the change. Please let me know what my next steps should be.

Reply:
hi looks good, thanks

Reply:
Should I click on 'Merge pull request'? It is my first time going through this procedure, so I'm a little unsure what I should do next to complete the change.

Reply:
After code 👍 you'll need a functional 👍 from QA or someone who tests the story before merging